vms/rpcchainvm/ghttp/http_client.go

```diff
@@ -160,7 +159,20 @@ func (c *Client) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	_, err = c.client.Handle(r.Context(), req)
+	reply, err := c.client.Handle(r.Context(), req)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	// Copy headers from the plugin process
+	header := w.Header()
+	clear(header)
+
+	for _, h := range reply.Header {
+		header[h.Key] = h.Values
+	}
```

Comment:
I need to think about if this is safe... upon closer inspection all of the header syncing in rpcchainvm feels weird.